Js/index.js

```diff
@@ -56,4 +56,30 @@ messageForm.addEventListener("submit", (e) => {
   newMessage.appendChild(removeButton);
   messageList.appendChild(newMessage);
   messageForm.reset();
-});
\ No newline at end of file
+});
+
+
+
+document.addEventListener('DOMContentLoaded', function () {
+  fetch('https://api.github.com/users/PromesseFongba/repos')
+  .then(response => response.json())
+  .then(repositories => {
+      const projectSection = document.getElementById('project'); 
+      const projectList = projectSection.querySelector('.year'); 
+
+      for (let i = 0; i < repositories.length; i++) {
```

Comment:
The for loop here is fine and works. However, I think using forEach would be better: 
  repositories.forEach((repo) => {
        const project = document.createElement("li");
        project.innerText = repo.name;
        project.className = "list";
        projectList.appendChild(project);
      });